mintapi/signIn.py

```diff
@@ -417,7 +414,10 @@ def sign_in(
 
 
 def home_page(driver):
-    element = driver.find_element(By.LINK_TEXT, "Sign in").click()
+    try:
+        element = driver.find_element(By.LINK_TEXT, "Sign in").click()
+    except WebDriverException:
+        logger.info("WebDriverException when clicking Sign In")
```

Comment:
If we get this exception, are we safe to continue? Or should we exit the run?

Reply:
Yes, it is safe to ignore and continue.